src/components/v2v3/V2V3Project/ProjectDashboard/components/JuiceCrowdRedirectModal.tsx

```diff
@@ -0,0 +1,61 @@
+import { Trans } from '@lingui/macro'
+import { Button, Modal } from 'antd'
+import ExternalLink from 'components/ExternalLink'
+import { useIsJuicecrowd } from 'hooks/v2v3/useIsJuiceCrowd'
+import Image from 'next/image'
+import { useState } from 'react'
+import { useProjectMetadata } from '../hooks/useProjectMetadata'
+import { getJuicecrowdUrl } from 'utils/juicecrowd'
+
+export function JuicecrowdRedirectModal() {
+  const { projectId } = useProjectMetadata()
+
+  const isJuicecrowd = useIsJuicecrowd()
+  const [open, setOpen] = useState<boolean>(isJuicecrowd)
+
+  if (!projectId) return null
+
+  return (
+    <Modal
+      open={open}
+      onCancel={() => setOpen(false)}
+      width={600}
+      okButtonProps={{ hidden: true }}
```

Comment:
@johnnyd-eth we can pass in `href` to this and I think it'll Just Work - no need for custom buttons